CHANGELOG.md

```diff
@@ -3,6 +3,42 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2024-05-28
+
+### Changes
+
+---
+
+Packages with breaking changes:
+
+ - There are no breaking changes in this release.
+
+Packages with other changes:
+
+ - [`storage_client` - `v2.0.2`](#storage_client---v202)
+ - [`realtime_client` - `v2.1.0`](#realtime_client---v210)
+ - [`supabase` - `v2.2.0`](#supabase---v220)
+ - [`supabase_flutter` - `v2.5.4`](#supabase_flutter---v254)
+
+Packages with dependency updates only:
+
+> Packages listed below depend on other packages in this workspace that have had changes. Their versions have been incremented to bump the minimum dependency versions of the packages they depend upon in this project.
+
+ - `supabase_flutter` - `v2.5.4`
+
+---
+
+#### `storage_client` - `v2.0.2`
+
```

Comment:
The changelog is missing here. I'm unsure what exactly happened on your site.

Reply:
Oh yeah. Let me just manually add it in.